app/static/app/js/components/ImagePopup.jsx

```diff
@@ -43,7 +44,7 @@ class ImagePopup extends React.Component {
     }
 
     imageOnError = () => {
-        this.setState({error: "Image is missing", loading: false});
+        this.setState({error: _("Image is missing"), loading: false});
```

Comment:
```suggestion
        this.setState({error: _("Image missing"), loading: false});
```
Either "The image is missing" or "Missing image/Image missing". Marked noun error.